profiles/common/modules/features/splash_screen/splash_screen.info

```diff
@@ -2,7 +2,7 @@ name = Splash screen
 description = Provide a homepage to choose language and store language in cookie.
 core = 7.x
 package = Multisite Features
-version = 7.x-2.0
+version = 7.x-2.02
 project = splash_screen
 features[features_api][] = api:2
 features[user_permission][] = administer splash screen
```

Comment:
Should we not add the variable 'splash_screen_homepage_hijacking'?
features[variable][] = splash_screen_homepage_hijacking